### DISCUSSION TITLE
[分享]绝区零助手，可以帮助清理日常

### DISCUSSION THREAD

**Discussion by yokuminto** [share121/inter-knot#4623] [科技]
![172162614731883e01eeca8a4b9e789541fa88139fa99724572ea5c2eb9824efbde80e4472bd0 0](https://github.com/user-attachments/assets/8650acee-a19a-43dd-892c-0bc7a2cf7b3c)
`作者近日陷入了星星大危机，急需绳匠点点星星
绝区零一条龙可以一键清理日常，刷刷空洞
基于图像识别、声音识别和一条龙框架的开源项目
这个是地址~
喜欢的小伙伴可以点点星星给作者鼓励鼓励下~
https://github.com/DoctorReid/ZenlessZoneZero-OneDragon

**Comment by FWJ321** [comment]
我去好东西

**Comment by ysdtal** [comment]
这个委托，我接了

**Comment by 206326402** [comment]
这个委托，我接了

**Comment by Robin-txdy** [comment]
好东西啊

**Comment by Junjun280** [comment]
我去，这的假的，真有啊，这不爽死了

**Comment by 1luik** [comment]
不会跑ai然后根游戏抢gpu或者cpu满吧
